### ISSUE TITLE
Relations not available inside the "where {}" block

### ISSUE THREAD

**Issue by cutalion** [rom-rb/rom-sql#247]
In rom-sql 2.0.0.rc `schema.relations` inside the `where {}` block is an empty hash.

```ruby
require 'rom'
require 'rom-sql'

config = ROM::Configuration.new(:sql, 'sqlite::memory')
conn = config.gateways[:default].connection

conn.create_table(:tasks) do
  primary_key :id
  column :status, String
end

config.relation(:tasks) do
  schema(:tasks, infer: true)

  def active
    where do
      ::Kernel.puts "REL: #{schema.relations.inspect}";
      schema.relations[:tasks][:status] == 'active'
    end
  end
end

rom = ROM.container(config)
rom.relations.tasks.active
#=> REL: {}
#=> where.rb:18:in `block in active': undefined method `[]' for nil:NilClass (NoMethodError)
```

**Comment by GustavoCaso** [comment]
Since the relations are store in the schema now I think we can close this one 🎉

**Comment by solnic** [comment]
This will be available in rom-sql 2.2.0 (done in master)
